client/app/index.jsx

```diff
@@ -8,17 +8,19 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      recs: new Array(4).fill({title: "a", description: "a", price: "a", photo_url: "http://lorempixel.com/640/480", _id: "a"}),
+      recs: [],
     };
   }
   componentDidMount() {
-    this.fetchRecommendations();
+    const pathName = window.location.pathname;
```

Comment:
At some point you'll need to refactor the rendering & id logic to conform to our standard for the proxy server